rdmd.d

```diff
@@ -64,7 +64,7 @@ int main(string[] args)
     //writeln("Invoked with: ", args);
     // Look for the D compiler rdmd invokes automatically in the same directory as rdmd
     // and fall back to using the one in your path otherwise.
-    string compilerPath = buildPath(dirName(thisExePath()), defaultCompiler);
+    string compilerPath = buildPath(dirName(thisExePath()), defaultCompiler ~ binExt);
```

Comment:
This is a bit of a problematic solution, because it "fixes" the issue in only one place (i.e. this line).

If the `defaultCompiler` should be different on Windows from on other OS's, then it should be `defaultCompiler` that is different: correctness shouldn't rely on the developer (elsewhere in the program) having to remember to append `binExt`.

Reply:
Well, there's the distinction between "the compiler filename" and "the compiler command". We generally care about the command (what do we need to pass to `spawnProcess` etc.), with the filename being this highly localized and specific exception.

Reply:
Yes, I suppose that's fair enough.  If the default compiler is `dmd` then, so long as `dmd.exe` is not actually in the same directory as the rdmd instance, any other executable `dmd` will do.  So it really is only for the fallback that the extension really matters.